packages/clay-css/src/scss/components/_date-picker.scss

```diff
@@ -0,0 +1,103 @@
+.date-picker {
+	.input-group-text {
+		@include clay-input-group-text-variant($date-picker-input-group-text);
+	}
+}
+
+// Date Picker Dropdown Menu
+
+.date-picker-dropdown-menu {
+	@include clay-dropdown-menu-variant($date-picker-dropdown-menu);
+}
+
+// Date Picker Nav
+
+.date-picker-nav {
+	@include clay-row($date-picker-nav-row);
+
+	.nav-btn {
+		@include clay-button-variant($date-picker-nav-btn);
+	}
+
+	.nav-btn-monospaced {
+		@include clay-button-variant($date-picker-nav-btn-monospaced);
+	}
+
+	select.form-control {
+		@include clay-select-variant($date-picker-nav-select);
+	}
+}
+
+.date-picker-nav-item {
+	align-items: center;
+	display: flex;
+	padding-left: 0.25rem;
+	padding-right: 0.25rem;
+}
+
+.date-picker-nav-item-expand {
+	flex-grow: 1;
+}
+
+.date-picker-nav-controls {
```

Comment:
I think the `date-picker-nav-controls` buttons should be grays, right?

Reply:
From what Lexicon has in Figma the buttons are more spaced with 8px between them.